### ISSUE TITLE
v0.12.2 - terraform plan on snowflake_stage keeps showing differences

### ISSUE THREAD

**Issue by iserko** [chanzuckerberg/terraform-provider-snowflake#194]
Since upgrading to v0.12.2 I've started getting diffs for all the stages I've added to Terraform.
Here's an example diff:
```
  ~ resource "snowflake_stage" "example" {
        database            = "DBNAME"
      + file_format         = "TYPE = JSON NULL_IF = ('[]')"
        id                  = "DBNAME|SCHEMANAME|STAGENAME"
        name                = "STAGENAME"
        schema              = "STAGENAME"
        storage_integration = "STORAGEINTEGRATIONNAME"
      + url                 = "s3://s3bucketname/data"
    }
```

tfstate after applying it
```
{
      "mode": "managed",
      "type": "snowflake_stage",
      "name": "STAGENAME",
      "provider": "provider.snowflake",
      "instances": [
        {
          "schema_version": 0,
          "attributes": {
            "aws_external_id": "",
            "comment": "",
            "copy_options": "",
            "credentials": null,
            "database": "DBNAME",
            "encryption": null,
            "file_format": "",
            "id": "DBNAME|SCHEMANAME|STAGENAME",
            "name": "STAGENAME",
            "schema": "SCHEMANAME",
            "snowflake_iam_user": "",
            "storage_integration": "STORAGEINTEGRATIONNAME",
            "url": ""
          },
          "private": "xxxyyy==",
          "dependencies": [
            "snowflake_database.database",
            "snowflake_schema.schema"
          ]
        }
      ]
    },
```

Even after applying ... it gives me the exact diff as I pasted on top.

Also ... it's strange that the url and file_format in the tfstate are just blanked out.

**Comment by brianhw** [comment]
We have just upgraded to 0.12.2 (from 0.11.0) and are also starting to see the same behavior, with both "url" and "file_format" properties.  They worked fine in 0.11.0, but now each time we plan, it wants to update these properties in our stages as well.   Is this something that is fixed by https://github.com/chanzuckerberg/terraform-provider-snowflake/pull/201?

**Comment by iserko** [comment]
confirmed fixed in 0.13.0 most likely due to #201
